### DISCUSSION TITLE
Show Code is not showing what I'd expect. What am I missing?

### DISCUSSION THREAD

**Discussion by smth** [storybookjs/storybook#15183] [Help] [answered]
Hi

When expanding the code panel in the Docs tab, I'm expecting to see something like his, but with some props:
![image](https://user-images.githubusercontent.com/1375708/121343601-c8323b80-c91a-11eb-8602-f274e27c75f1.png)

I'm getting this however:
![image](https://user-images.githubusercontent.com/1375708/121343686-ded89280-c91a-11eb-9d0a-6c03c79e5b85.png)

I'm using the example stories generated by `npx sb init` in a Vue project.

What am I missing?

**Comment by shilman** [comment, marked as answer]
@smth you're probably missing this https://github.com/storybookjs/storybook/issues/13917
